server/src/main/java/org/opensearch/action/bulk/TransportShardBulkAction.java

```diff
@@ -146,6 +147,7 @@ public class TransportShardBulkAction extends TransportWriteAction<BulkShardRequ
      * term validation in presence of a new primary.
      */
     private final String transportPrimaryTermValidationAction;
+    private final Tracer tracer;
```

Comment:
@rayshrey please remove `tracer` , it is provided by `TransportWriteAction`

Reply:
Done